bin/solr/update_document

```diff
@@ -27,6 +28,44 @@ def get_solr_id_from_db_id(db_id)
   return nil
 end
 
+def update_document_type_form(db_id, solr_id)
+  # TODO(asher): Refactor this and the code in bin/cron/process_forms to a
+  # shared method.
+  db_form = DB[:forms].where(id: db_id).first
+
+  kind = Object.const_get(db_form[:kind])
+  begin
+    solr_doc = JSON.load(db_form[:data]).
```

Comment:
Should we name this solr_form_doc to be more consistent with the update user method?